components/AnimatedRegion.js

```diff
@@ -1,45 +1,38 @@
 /* eslint-disable */
+import {Animated} from 'react-native';
 
-class AnimatedRegion extends AnimatedWithChildren {
-  // latitude: AnimatedValue;
-  // longitude: AnimatedValue;
-  // latitudeDelta: AnimatedValue;
-  // longitudeDelta: AnimatedValue;
-  // _listeners: {[key: string]: {
-  //  latitude: string,
-  //  longitude: string,
-  //  latitudeDelta: string;
-  //  longitudeDelta: string,
-  // }};
+const AnimatedWithChildren = Object.getPrototypeOf(Animated.ValueXY);
+if (AnimatedWithChildren.name !== 'AnimatedWithChildren') console.error('AnimatedRegion could not obtain AnimatedWithChildren base class');
```

Comment:
I made a release build but I couldn't get it to trigger the error.
Still, I think you're right and we should stay on the safe side here and wrap it in a __DEV__